provider/resource_keycloak_openid_client.go

```diff
@@ -12,6 +12,7 @@ import (
 var (
 	keycloakOpenidClientAccessTypes                        = []string{"CONFIDENTIAL", "PUBLIC", "BEARER-ONLY"}
 	keycloakOpenidClientAuthorizationPolicyEnforcementMode = []string{"ENFORCING", "PERMISSIVE", "DISABLED"}
+	keycloakOpenidClientPkceCodeChallengeMethod            = []string{"", "plain", "S256"}
```

Comment:
Is the empty string needed here if the `pkce_code_challenge_method` attribute is optional?  I would just expect users to omit this entirely rather than provide an empty string.

Reply:
I'm not sure about this. Would it still be possible to omit this value if the actual value is provided by a variable via interpolation syntax?

Reply:
I have removed the empty string from the options.